<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QWidget" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MacrosHub - Настройки</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../imgs/logo_apple.png</normaloff>../imgs/logo_apple.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#SettingsWindow{background-color:#fff;}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <property name="spacing">
    <number>20</number>
   </property>
   <item row="4" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QFrame" name="f_language">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>100</number>
      </property>
      <property name="rightMargin">
       <number>100</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="l_language">
        <property name="font">
         <font>
          <family>Montserrat Medium</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#3E3E3E;</string>
        </property>
        <property name="text">
         <string>Язык</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="rb_ru">
        <property name="font">
         <font>
          <family>Montserrat</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton#rb_ru{color:#3E3E3E;}
QRadioButton#rb_ru::indicator{background-color:#fff;width:18px;height:18px;border-radius:15px;}
QRadioButton#rb_ru::indicator:unchecked{border:6px solid #D9D9D9;}
QRadioButton#rb_ru::indicator:checked{border:6px solid #7CC8FF;}</string>
        </property>
        <property name="text">
         <string>Русский</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="rb_en">
        <property name="font">
         <font>
          <family>Montserrat</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton#rb_en{color:#3E3E3E;}
QRadioButton#rb_en::indicator{background-color:#fff;width:18px;height:18px;border-radius:15px;}
QRadioButton#rb_en::indicator:unchecked{border:6px solid #D9D9D9;}
QRadioButton#rb_en::indicator:checked{border:6px solid #7CC8FF;}</string>
        </property>
        <property name="text">
         <string>English</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="btn_apply">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Medium</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btn_apply{color:#fff;border-radius:25px;background-color:#7CC8FF;}
QPushButton#btn_apply::hover{color:#fff;border-radius:25px;background-color:#52A9FF;}</string>
     </property>
     <property name="text">
      <string>Применить</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QFrame" name="f_theme">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>100</number>
      </property>
      <property name="rightMargin">
       <number>100</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="l_theme">
        <property name="font">
         <font>
          <family>Montserrat Medium</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#3E3E3E;</string>
        </property>
        <property name="text">
         <string>Тема</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="rb_light">
        <property name="font">
         <font>
          <family>Montserrat</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton#rb_light{color:#3E3E3E;}
QRadioButton#rb_light::indicator{background-color:#fff;width:18px;height:18px;border-radius:15px;}
QRadioButton#rb_light::indicator:unchecked{border:6px solid #D9D9D9;}
QRadioButton#rb_light::indicator:checked{border:6px solid #7CC8FF;}</string>
        </property>
        <property name="text">
         <string>Светлая</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="rb_dark">
        <property name="font">
         <font>
          <family>Montserrat</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton#rb_dark{color:#3E3E3E;}
QRadioButton#rb_dark::indicator{background-color:#fff;width:18px;height:18px;border-radius:15px;}
QRadioButton#rb_dark::indicator:unchecked{border:6px solid #D9D9D9;}
QRadioButton#rb_dark::indicator:checked{border:6px solid #7CC8FF;}</string>
        </property>
        <property name="text">
         <string>Темная</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
